project/src/PlayerCamera.gd

```diff
@@ -0,0 +1,14 @@
+extends Camera2D
+
+
+export var update_period := 15
+
+func _physics_process(delta: float) -> void:
```

Comment:
I actually think this could read:
```
if is_update_frame():
    Signals.emit_signal(...)
```
And remove lines 13 and 14. What do you think?